src/views/Summary.js

```diff
@@ -15,25 +16,45 @@ function Summary({ selectedMonth, filteredExpenses }) {
     setTotalExpenses(expenseSum);
   }, [filteredExpenses]);
 
+  // Calculate spending rate deviation
+  useEffect(() => {
+    const date = new Date(selectedMonth);
+
+    // Get last day of selected month
+    const daysInMonth = new Date(
+      date.getFullYear(),
+      date.getMonth() + 1,
+      0
+    ).getDate();
+
+    // Set day equal to today's date or last day of the month
+    const dayOfMonth =
+      selectedMonth === currentMonth ? new Date().getDate() : daysInMonth;
+
+    const targetRate = totalBudget / daysInMonth;
+    const actualRate = totalExpenses / dayOfMonth;
+    const deviation = (actualRate - targetRate) / targetRate;
+    setSpendingRateDeviation(deviation);
+  }, [currentMonth, selectedMonth, totalBudget, totalExpenses]);
+
   return (
     <div className="card h-100">
       <h3 className="card-header">
-        Summary for{' '}
-        <span className="text-primary fw-bold">{selectedMonth}</span>
+        Summary for <span className="fw-bold">{selectedMonth}</span>
       </h3>
       <div className="card-body container d-flex flex-column justify-content-between">
         <h4 className="fw-light row">
           <span className="col-6 col-md-4 col-lg-6 col-xl-5">
             Starting budget:
```

Comment:
i am not sure if it would help the readability if we do not wrap items which are as short as this one. what do you think?

Reply:
My feeling is that, if we don't wrap these particular items, we won't be able to control how much spacing there is between the "labels" and the money amounts in the grid.

Reply:
ah, I did not make myself clear. I only meant the line breaks.

Reply:
Ah, gotcha. We decided to use Prettier to ensure consistent formatting. I think the config we are using has max col width 80.

Reply:
ah understand. I typically disable the line length, because i think it is not contemporary anymore.